app/services/open_library_service.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class OpenLibraryService
+  require 'HTTParty'
+
+  BASE_URI = OPEN_LIBRARY['base_uri']
+
```

Comment:
you can include a `initialize` method and not use the prefix self in all methods

```ruby
  # How to use thi service
  OpenLibraryService.new(params[:isbn]).get_library

  # iniaitlize method receive the isbn and you can use inside the class
  def initialize(isbn)
    @isbn = isbn
  end
```